### PR TITLE
add metadata to dns switcher plugin

### PR DESCRIPTION
**Pull request by saiqulhaq**

### CODE REVIEW

File: Network/dnsswitcher.1d.sh [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 # DNS Switcher
-# BitBar plugin
+# The list of DNS options should be defined on this file
 #
-# by M Saiqul Haq
+# <bitbar.title>DNS Switcher</bitbar.title>
+# <bitbar.version>v1.4</bitbar.version>
+# <bitbar.author>M Saiqul Haq</bitbar.author>
+# <bitbar.author.github>saiqulhaq</bitbar.author.github>
+# <bitbar.desc>Switch DNS to your defined DNS options</bitbar.desc>
```
Comment (keithamus) [inline Network/dnsswitcher.1d.sh]: Could you please add an image? That'd be sweeeeet.

Comment (saiqulhaq) [inline Network/dnsswitcher.1d.sh]: do you mean screenshot of this plugin?

Comment (keithamus) [inline Network/dnsswitcher.1d.sh]: Yes exactly! The metadata is `<bitbar.image>`. [See here](https://github.com/matryer/bitbar#metadata)

Comment (saiqulhaq) [inline Network/dnsswitcher.1d.sh]: OK, will do

**Pull request merged**